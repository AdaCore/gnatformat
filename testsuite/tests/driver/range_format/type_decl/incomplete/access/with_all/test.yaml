description: ConcreteTypeDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 77",
    "-EL 78",
    "--pipe",
  ]
