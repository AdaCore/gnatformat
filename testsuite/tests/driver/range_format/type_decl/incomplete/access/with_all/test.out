test.ads(<ConcreteTypeDecl ["Car_Name"] test.ads:77:4-77:45>) - 77:1-77:45
^
   type Car_Name is access all Car'Class;$

